installing/installing_azure_stack_hub/installing-azure-stack-hub-user-infra.adoc

```diff
@@ -83,7 +83,7 @@ include::modules/installation-user-infra-generate-k8s-manifest-ignition.adoc[lev
 
 [role="_additional-resources"]
 .Additional resources
-* xref:../../installing/installing_azure/manually-creating-iam-azure.adoc#manually-create-iam_manually-creating-iam-azure[Manually creating long-term credentials]
+* xref:../../installing/installing_azure/installing-azure-customizations.adoc#manually-create-iam_installing-azure-customizations[Manually creating long-term credentials]
```

Comment:
Replaced a defunct link, but not entirely sure this should be here. Asked [here](https://github.com/openshift/openshift-docs/pull/64288/files#r1358695929).